devDocs/developerGuide.t2t

```diff
@@ -208,6 +209,30 @@ This will all be covered in depth later.
 NVDA loads an App Module for an application as soon as it notices the application is running.
 The App Module is unloaded once the application is closed or when NVDA is exiting.
 
+++ Associating app Module with an executable ++
+As explained above sometimes the default way of associating app Module with an application is not flexible enough. Examples include:
+- You want to use a single app Module for various binaries (perhaps both stable and preview version of the application should have the same accessibility enhancements)
+- The executable file is named in a way which conflicts with the Python naming rules i.e. application named "time" which conflicts with the built-in module from the standard library
```

Comment:
Trying to make this one read more clearly. Not sure if this does it, but here's an attempt.

```suggestion
- The executable file is named in a way which conflicts with the Python naming rules. I.e. for an application named "time", naming the app Module "time.py" would conflict with the built-in module from the standard library
```